docs/faq/faq-felizandfable.md

```diff
@@ -0,0 +1,75 @@
+In a nutshell Fable.React and Feliz are two F# libraries which perform a similar function:
+
+* Provide a typesafe F# wrapper to allow you to interact with React.
+* Provide a way to create your own wrappers around existing React components.
+* Provide a DSL for you to consume and create React wrappers in a consistent way.
+
+## DSL differences
+The main distinction between the two libraries is that Fable.React follows a layout as follows:
+
+```fsharp
+element [ attribute; attribute ] [ childElement; childElement ]
+```
+
+> Note: Code snippets below are for illustrative purposes only and do not compile.
+
+For example:
+
+```fsharp
+h1 [ style "color:Tomato" ] [
+    p [] [ text "Hello" ] // no attributes
+    p [] [ text "Another paragraph" ] // no attributes
+    h2 [ style "color:Blue" ] [] // no child elements
+]
+```
```

Comment:
I don't think this is a good idea to show invalid code.

I understand that you don't want to show complexe code, but I think using `ClassName <name-of-class>` is simple enough compared to what `Style` would looked like.

Reply:
Agreed, I simply was writing the code directly in GH. I'll update with valid code.